appveyor.yml

```diff
@@ -9,42 +9,22 @@ environment:
           MINICONDA_DIRNAME: C:\Miniconda36-x64
         - PYTHON_VERSION: 3.7
           MINICONDA_DIRNAME: C:\Miniconda37-x64
-          
+
     ANACONDA_TOKEN:
         secure: nxF/a2f3iS9KXGu7B/wKJYAk7Sm5wyAjoZoqJvPbRoVK4saaozVwOxDrjwJjJAYb
 
 install:
     - set "PATH=%MINICONDA_DIRNAME%;%MINICONDA_DIRNAME%\\Scripts;%PATH%"
     - conda config --set always_yes yes
     - conda update -q conda
-    - conda create --quiet --name cqtest -c cadquery -c conda-forge -c dlr-sc pythonocc-core=0.18.2 python=%PYTHON_VERSION% pyparsing mock coverage codecov
-    - activate cqtest
-    - pip install .[dev]
-    - python setup.py install
+    - conda env create -f environment.yml
+    - activate cadquery
+    - conda install -c conda-forge -c defaults -c cadquery python=%PYTHON_VERSION%
```

Comment:
Same here, should it be installing cadquery=2 from the channel?

Reply:
No, cadquery should be installed from the current dir (and this is precisely what the new environment.yml does)